src/main/java/net/sf/jabref/MetaData.java

```diff
@@ -40,37 +41,35 @@
 import net.sf.jabref.logic.labelpattern.DatabaseLabelPattern;
 import net.sf.jabref.logic.util.strings.StringUtil;
 import net.sf.jabref.model.database.BibDatabaseMode;
-import net.sf.jabref.sql.DBStrings;
 
+import com.google.common.eventbus.EventBus;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 
 public class MetaData implements Iterable<String> {
 
     private static final Log LOGGER = LogFactory.getLog(MetaData.class);
     public static final String META_FLAG = "jabref-meta: ";
-    private static final String SAVE_ORDER_CONFIG = "saveOrderConfig";
+    public static final String SAVE_ORDER_CONFIG = "saveOrderConfig";
 
-    private static final String SAVE_ACTIONS = "saveActions";
-    private static final String PREFIX_KEYPATTERN = "keypattern_";
-    private static final String KEYPATTERNDEFAULT = "keypatterndefault";
-    private static final String DATABASE_TYPE = "databaseType";
+    public static final String SAVE_ACTIONS = "saveActions";
```

Comment:
Done. The synchronizer now uses an sql table with columns `META_KEY` and `META_VALUE`.

> Are there any problems in storing metadata in this form?
No, but it is an unusual way to do this with relational database system. But it works properly.